.github/actions/common/artifact_utils.py

```diff
@@ -9,7 +9,9 @@
 def add_common_args(parser: argparse.ArgumentParser):
     parser.add_argument('-s', '--commit_sha', help='Commit hash for which artifacts were generated', required=True)
     parser.add_argument('-b', '--branch_name', help='Name of GitHub branch', required=False,
-                        default=os.getenv('GITHUB_BASE_REF') or os.getenv('GITHUB_REF_NAME'))
+                        default=os.getenv('GITHUB_BASE_REF') or
```

Comment:
please add the logic to remove 'refs/heads/' for user's branch too

Reply:
It doesn't make much sense, since user-passed branch is usually hard-coded and doesn't include refs/ prefixes. If we'll have some problems with that in the future - I'll make this update